### ISSUE TITLE
Any plans to support i18n?

### ISSUE THREAD

**Issue by clemlatz** [mpigsley/keyforge-arena#16]
Are you planning to support other languages than english? If so I'd be happy to help with french translation. But it's something you'd wanna plan as early as possible.

**Comment by mpigsley** [comment]
Yeah, I would love to! The site is in some super early development but that's a perfect time to start thinking about this.

I've done this with another open source project and I used a google doc to help manage all the translations. I'll get that wired up at some point soon and send you that link. It might be a bit before all that happens though.

**Comment by clemlatz** [comment]
Great! I'll be waiting for your link, then.

Also, you would ideally include card images with localized texts. Not sure how you got images in the bootstrap directory, but french images (and I guess every other languages) are available from the official site by changing 'en' to 'fr' in the url.

![english](https://cdn.keyforgegame.com/media/card_front/en/341_1_7C854VPW72RH_en.png) ![french](https://cdn.keyforgegame.com/media/card_front/fr/341_1_7C854VPW72RH_fr.png)

https://cdn.keyforgegame.com/media/card_front/en/341_1_7C854VPW72RH_en.png
https://cdn.keyforgegame.com/media/card_front/fr/341_1_7C854VPW72RH_fr.png

**Comment by mpigsley** [comment]
I pulled them from the main site so I'll do that for French as well. Thanks for the help!
